composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
```

Comment:
> I don't like fetching some deps by Composer and some not.

As you want but IMHO, it's more a tool than a "dependency".

> You don't like it? Cool. You are not enforced to download dev deps.

Not really true for all projects. Some can require dev packages other than phpunit and we can select which one we want to install.

By installing with composer, you get dependencies **described** by phpunit, not locked by `composer.lock` like for `.phar`. This mean you increase the issue risk to get PHPUnit not working because of a sub-dependency update.

So this is not only a matter of taste for me. :wink: 

> Composer is nicer way to describe required version. like not supporting 4.1

Well, if you not support it, tests will fail. :-)